views/main.js

```diff
@@ -132,28 +132,35 @@ function render(data) {
     $answers.html('');
 
   // Back Button
-  if (actions.length) {
-    var $backBtn = $('<button class="back-button">Back</button>');
-    $backBtn.click(function() {
-      var previous = actions.pop();
-
-      var checklistIdx = checklist.indexOf(previous.checklist);
-      if (checklistIdx >= 0) {
-        checklist.splice(checklistIdx, 1);
-      }
-
-      render(previous);
-    });
+    if (actions.length) {
+	var $backBtn = createBackButton();
     $back.html($backBtn);
   } else {
     $back.html('');
   }
 }
 
+function createBackButton() {
```

Comment:
New createBackButton() function. You can call it anytime you want a back button.